packages/flutter_tools/lib/loader.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
```

Comment:
This file could also live in an internal implementation directory in the flutter package - then you wouldn't need to exclude the file from analysis.